Documentation/Performance/PerfGettingStarted.md

```diff
@@ -0,0 +1,79 @@
+# Performance
+
+## Getting Started
+
+If you are new to Unity performance analysis, there are some great resources that Unity provides that
+walks you through the general thought process of how to conduct a performance analysis, along with common
+pitfalls and recommendations. As other resources will say, there is no magic solution for performance issues
+– every situation is unique – and digging into performance issues involves an iterative process where you
+typically identify a problematic scenario, measure the problem (using some sort of profiling tool),
+dig into the hotspots (using that profiling tool), test out a fix to that hotspot, and then repeat
+until performance is within desired bounds.
+
+### [Introduction to Unity performance optimization for beginners](https://www.youtube.com/watch?v=1e5WY2qf600)
```

Comment:
links as document section headers seems a bit unusual. 

perhaps move the link to the word "video" in the following sentence

Reply:
I can just make these bold instead. I was looking to give each link a title + description, which this accomplishes.